sample/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,6 @@
     <string name="backend_error_info">Ups, something went wrong</string>
     <string name="app_title">Stream Chat</string>
     <string name="log_out_title">Log out</string>
-</resources>
\ No newline at end of file
+    <string name="new_messages_single">%s New Message</string>
+    <string name="new_messages_plural">%s New Messages</string>
```

Comment:
If we use plurals we won't be able to configure this using XML. We can use plurals to configure the View programatically, but then the XML configuration and code configuration would be different, which I don't consider to be ideal.

Reply:
Hmm, so we can't get plural strings as the parameter for an XML attribute?

Reply:
No, only using code.

Reply:
Well, it is possible to do it using databinding: https://stackoverflow.com/questions/17458965/getting-a-string-from-plurals-to-use-in-xml

But I prefer not to depend on it.

Reply:
I found [this ugly way](https://stackoverflow.com/a/25648349) of parsing whether a value you got in the attribute is a plural, but it really doesn't seem like it would be worth it. Let's go with just single vs plural for now then 👍